monitoring/shutdown-javaagent/src/main/java/com/google/cloud/runtimes/monitoring/shutdown/log/ILogging.java

```diff
@@ -0,0 +1,12 @@
+package com.google.cloud.runtimes.monitoring.shutdown.log;
+
+public interface ILogging {
```

Comment:
Javadoc.

Also, I would recommend adding javadoc for each method to make it clear how to implement this interface.